IshiHisashi/GlucoFit
# feat : JDEV-51 build a auth mock into test page

## Summary
This is continuous of auth functionality. Since this is a mobile app, JWT token should be stored in secure storage built by expo. I have created util functions to manage token on the client and made a simple mock on the test page.

## Changes
Related files are put in `/frontend/src/testPages/auth_temp`.
- `utilAuth.ts` : handlers to manage token generated from server. This will be used further development.
- `authMock.tsx` : this is just for this test purpose. will be replaced.

## How to test
Place you network IP address on `const IP = "" ` in line#13 of `authMock.tsx`.

Go to Test tab
![IMG_2016](https://github.com/user-attachments/assets/a0f0facc-27dc-4b50-89a3-2594aa88c4ee)
And input email and password (whatever you like), then press 'sing up'. 
If you go to MongoDB console, you will see a new user doc is created based on it.

Then, try login next. 

When each of above two operation is succeeded, you will see pop up to tell you it's done.
